### PR TITLE
Workspace conversion

### PR DESCRIPTION
**Pull request by efredriksson-modelon**
### How to test:
You can test it by running:
 * poetry install
 * poetry shell
 * And then execute the Python code:
```
from modelon.impact.client import Client

c = Client('<server URL>')
workspace_id = '<workspace ID>'
workspace = c.convert_workspace(workspace_id).wait()
print(workspace.definition.name)
```
on a workspace that should be converted.

### CODE REVIEW

**Comment by efredriksson-modelon** [comment]
@abhilash-kumar-nair Sure, note that this one is WIP for now and that I have not added tests yet 😃

**Comment by efredriksson-modelon** [comment]
@abhilash-kumar-nair Now it is all ready 😃

File: modelon/impact/client/operations/workspace/conversion.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
+from modelon.impact.client import exceptions
+from modelon.impact.client.entities.workspace import Workspace, WorkspaceDefinition
+from modelon.impact.client.operations.base import AsyncOperation, AsyncOperationStatus
+from modelon.impact.client.sal.service import Service
+
+
+class WorkspaceConversionOperation(AsyncOperation):
+    """
+    An export operation class for the modelon.impact.client.entities.workspace.
```
Comment (abhilash-kumar-nair) [inline modelon/impact/client/operations/workspace/conversion.py:9]: export --> conversion

**Review by abhilash-kumar-nair — COMMENTED** [review]
Really nice to have the conversion workflow implemented with the client. Just a comment/question, how would the user know he needs to convert, would that be a failed GET /workspace_id call that indicated that the user needs to call convert? I was thinking if we had some means to know if a conversion is needed and raise appropriate exceptions for get_workspace() OR automatically do the conversion as a part of the get_workspace() call?

**Comment by efredriksson-modelon** [comment]
> Really nice to have the conversion workflow implemented with the client. Just a comment/question, how would the user know he needs to convert, would that be a failed GET /workspace_id call that indicated that the user needs to call convert? I was thinking if we had some means to know if a conversion is needed and raise appropriate exceptions for get_workspace() OR automatically do the conversion as a part of the get_workspace() call?

You will get
bild

which should make it clear that the workspace needs to be converted. I think having this as a separate step and not something that happens automatically is a feature as the user might not want to convert their workspace.

**Review by abhilash-kumar-nair — APPROVED** [review]
Makes sense! 👍

**Pull request merged**